src/core/stdc/config.d

```diff
@@ -164,7 +164,9 @@ else version (Posix)
   }
 }
 
-version (CRuntime_Microsoft)
+version (LDC)
+    alias c_long_double = real; // 64-bit real for MSVC targets
+else version (CRuntime_Microsoft)
```

Comment:
Could you move all compiler version conditions above CRuntime_Microsoft?

Reply:
I take it GDC will never have MSVC support, just MinGW on Windows?

Reply:
Done.